packages/syft/src/syft/client/domain_client.py

```diff
@@ -197,6 +198,35 @@ def code_history(self) -> CodeHistoriesDict:
     def code_histories(self) -> UsersCodeHistoriesDict:
         return self.api.services.code_history.get_histories()
 
+    @property
+    def images(self) -> Optional[APIModule]:
+        if self.api.has_service("worker_image"):
+            return self.api.services.worker_image
+        return None
+
+    @property
+    def worker_pools(self) -> Optional[APIModule]:
+        if self.api.has_service("worker_pool"):
+            return self.api.services.worker_pool
+        return None
+
+    @property
+    def workers(self):
+        """
+        TODO: We will work on this later with a new, separate WorkerStash
+        """
+        result = self.api.services.worker_pool.get_all()
+        if isinstance(result, DictTuple):
+            worker_names = ""
+            image_tag_workers = {}
+            for worker_pool in result:
+                for worker in worker_pool.workers:
+                    worker_names += f"{worker.name}\n"
+                image_tag_workers[worker_pool.syft_worker_image_name_tag] = worker_names
+            return DictTuple(image_tag_workers)
+        else:
+            return SyftError(message=f"{result.err()}")
```

Comment:
It makes sense that `client.workers()` does not print out the `_repr_html_`. Plain `DictTuple` objects do not have the `_repr_html_` method.

Also if we're not using this right now I think it's better to remove it from this PR. We can add it back when we actually use it.

Reply:
Done